### ISSUE TITLE
add score function for multinomial

### ISSUE THREAD

**Issue by jasmainak** [glm-tools/pyglmnet#97]
'multinomial' is typically evaluated using a score like accuracy or F1: deviance and pseudo_R2 are not common.

cc @pavanramkumar

**Comment by daniel-acuna** [comment]
There is no clear way of measuring the performance of a multiclass classifier. I like log likelihood ratio between a null model and full model but I agree it is not standard.

**Comment by jasenjackson** [comment]
Could we measure performance with the sci-kit log_loss function? 

http://scikit-learn.org/stable/modules/model_evaluation.html#log-loss

I was looking through their documentation, and it says that logistic regression loss is a common way of evaluating performance for multinomial classifiers and neural networks.

**Comment by pavanramkumar** [comment]
closing this since we will track these issues while implementing a separate `MultinomialGLM()` class.
